usecases/recording.cameras.opencv.3/docker-compose.yml

```diff
@@ -29,24 +29,21 @@ services:
         build: .
         image: seresearch/opendavinci-ubuntu-16.04-complete:latest
         volumes:
-        - .:/opt/data
-        command: "/opt/od4/bin/odsupercomponent --cid=${CID} --verbose=1  --configuration=/opt/data/configuration"
+        - .:/opt/recordings
```

Comment:
The configuration file shall not be mapped to the recordings folder to avoid manually copying it. Could you please have a look at test-ps3controller to have the similar mapping for the configuration file?